docs/providers.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Providers
+icon: book-sparkles
+---
+
+Connect pulls data from `Providers`. `Providers` utilize public REST APIs and/or websockets from either centralized or decentralized exchanges. Below is a list of all available `Providers` in Connect.
+
+# Centralized Exchange Providers
+### REST API
+- binance_api
+- bitstamp_api
+- coinbase_api
+- coingecko_api
+- gecko_terminal_api
```

Comment:
Idk if we should include providers like `gecko_terminal` and `coingecko` in these lists as they are not primary sources and should not be used in any production oracle

Reply:
[remove the gecko](https://github.com/skip-mev/slinky/pull/604/commits/296e19f3cb0b2c5a1461d7eecf807102dcd2c510)